pkg/kubecost/asset_json.go

```diff
@@ -477,6 +477,9 @@ func (n *Node) MarshalJSON() ([]byte, error) {
 	jsonEncodeString(buffer, "end", n.End.Format(time.RFC3339), ",")
 	jsonEncodeFloat64(buffer, "minutes", n.Minutes(), ",")
 	jsonEncodeString(buffer, "nodeType", n.NodeType, ",")
+	if n.Pool != "" {
+		jsonEncodeString(buffer, "pool", n.Pool, ",")
```

Comment:
Question: What happens if n.Pool is empty? 

Shouldnt json encoding handle the case of an empty n.Pool or is your idea not to present n.Pool if it's not there?